testdata/zk/Cheatcodes.t.sol

```diff
@@ -120,6 +129,11 @@ contract ZkCheatcodesTest is DSTest {
 
     address constant TEST_ADDRESS = 0x6Eb28604685b1F182dAB800A1Bfa4BaFdBA8a79a;
 
+    bytes32 constant STORAGE__INITIALIZABLE = bytes32(uint256(0));
+
+    bytes32 constant TRUE = bytes32(uint256(1));
+    bytes32 constant FALSE = bytes32(uint256(0));
```

Comment:
These `TRUE` and `FALSE` are also incorrect. The false is the default zero-value you get and the `TRUE` is the `initialized = true` you did encoded. You can have a simpler behavior by using the slotrage slot `0` as `u8` and `slot = 0xff`